src/golem_network/golem_network.ts

```diff
@@ -98,7 +98,7 @@ export class GolemNetwork {
    * console.log(status);
    * ```
    */
-  public async createJob<Output = unknown>(worker: Worker<unknown, Output>) {
+  public async createJob<Output = unknown>(worker: Worker<Output>) {
```

Comment:
I think that we could drop `= unknown` as the type inference would work here just perfectly.